File: olderFiles/stang/objective_c/pierwsza/pierwsza/ViewController.m
//
//  ViewController.m
//  pierwsza
//
//  Created by SG on 2016-09-14.
//  Copyright © 2016 SG. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *etykieta;
@property (weak, nonatomic) IBOutlet UIButton *przycisk;
@property (weak, nonatomic) IBOutlet UITextField *poletekstowe;
- (IBAction)klik:(id)sender;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)klik:(id)sender {
    
    NSString *wczytanytext = self.poletekstowe.text;
    NSString *powitanie = [NSString stringWithFormat:@"Hello %@ !",wczytanytext];
    
    self.etykieta.text = powitanie;
}
@end

File: stang/first/sw_ex_1/sw_ex_1/ViewController.swift
//
//  ViewController.swift
//  sw_ex_1
//
//  Created by SG on 2016-09-30.
//  Copyright © 2016 SG. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    
    @IBOutlet weak var textSender: UITextField!
    
    @IBOutlet weak var textReceiver: UILabel!
    
    
    @IBAction func btnTransformer(sender: AnyObject) {
        textReceiver.text = textSender.text
    }
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

